app/src/main/res/layout/activity_main.xml

```diff
@@ -5,6 +5,7 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:fitsSystemWindows="true"
```

Comment:
This is for android15/sdk35. There's now a default display mode called "edge-to-edge" which will draw windows to the edge (sometimes below system UI elements). Presumably this has some utility, but for this app it just serves to obscure our toggle button on newer android devices.

Using fitsSystemWindows seems to fix it. (I hope)